package.json

```diff
@@ -20,5 +20,8 @@
     "@divriots/studio-doc-compiler": "^0.0.31",
     "lit": "^2.0.0",
     "twind": "^0.16.16"
+  },
+  "exports": {
+    "./label/define": "./label/dockit-label.define.js"
```

Comment:
once you define `exports`, all other files will not be accessible, you kinda lock file system import paths
did you try `bkl link` with this branch and use it in some project to see whether everything works?

Reply:
You're right. It doesn't work. I'll remove the define from this PR so that everything gets exported as before and we'll do a new PR with this later.

Reply:
You can use this but for backwards compatibility you can do:
```js
{
  "exports": {
    "./": "./"
  }
}
```
which will re-enable regular "file-system imports", this is often done to add export fields without doing a breaking change (but good idea to deprecate and break it in the next major, or minor if <1.0.0).